src/features/SolarSystem/SolarRender.h

```diff
@@ -0,0 +1,345 @@
+#pragma once
+
+#include <Renderer.h>
+#include <gl\GL.h>
+#include <gl\GLU.h>
+#include"features\SolarSystem\SolarResource.h"
+namespace Features
+{
+	using namespace Interfaces;
+
+	class SolarSystemRenderer : public IRenderer
+	{
+	private:
+		float		g_Angle;
```

Comment:
As a generic guideline, can we use GL types?

e.g. in this case, can this be changed to GLfloat?

Reply:
I accepted your suggestion, Changed it .